src/main/java/step4/domain/RacingCar.java

```diff
@@ -67,12 +67,12 @@ public List<String> getWinner() {
         return winner;
     }
 
-    public void startRacing(OutputView outputView) {
-        for (int i = 0; i < numberOfRace; i++) {
-            doRacing();
-            RacingCarResultView.printRacingCarStatus(outputView, this);
-        }
-        RacingCarResultView.printRacingCarWinner(outputView, this);
+    public Integer compareNumberOfRace(int compareNumber) {
+        if (this.numberOfRace > compareNumber)
```

Comment:
```suggestion
        return Integer.compare(this.numberOfRace, compareNumber);
```

단순 값비교라면 위처럼 바꿀 수 있습니다.

그런데, 이 구조보단 RacingCar 가 횟수를 가지고 있으니 횟수만큼 반복해서 racing 을 하는 책임을 할당하는게 더 나을것 같아요

Reply:
감사합니다 ! 생각 못했던 부분입니다 ㅎㅎ